docs/streams/upgrade-guide.html

```diff
@@ -151,6 +151,13 @@ <h3><a id="streams_api_changes_420" href="#streams_api_changes_420">Streams API
         More details can be found in <a href="https://cwiki.apache.org/confluence/x/jQobFw">KIP-1221</a>.
     </p>
 
+    <p>The streams thread metrics <code>commit-ratio</code>, <code>process-ratio</code>, <code>punctuate-ratio</code>, and <code>poll-ratio</code> have been updated.
```

Comment:
This fix wont' get into AK 4.2, but only AK 4.3. We are passed code-freeze and only blockers can get merged now.

We need to start a new section for 4.3 and move this change there.